_posts/2016-07-11-value-of-automated-e2e-testing.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+author: riikkanen
+title: >
+  The Value of Automated e2e-testing in a Software Project
+excerpt: >
+  A peek into automated e2e-testing done in a software project. The first try with Wicket Page Tester. The second and more sophisticated version with JBehave and Selenium.
```

Comment:
"A peek into automated e2e-testing done in a software project." -> "done" can be left out.